### ISSUE TITLE
Requires Qt6

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-desktop#1300]
Secure that Qt6 is required to compile. If yomeone still want to compile with Qt5 it will have tomset proper env variable. README should be also updated.

**Comment by kelson42** [comment]
@adamlamar doable?

**Comment by adamlamar** [comment]
Yes it should be possible to [require a minimum Qt version](https://stackoverflow.com/questions/59091049/specify-minimum-qt-version-for-qtcreator-qmake-build) unless an environment variable is set to override the minimum.

**Comment by kelson42** [comment]
We will remove support of Qt5 after release of version `2.5.0`.

**Comment by kelson42** [comment]
@adamlamar Just FYI, now that `2.5` has been released, we will implement this quickly as part of `2.6.0` (which should be a pretty quick milestone).
